certification/internal/shell/validate_operator_bundle.go

```diff
@@ -5,20 +5,29 @@ import (
 	"strings"
 
 	"github.com/komish/preflight/certification"
+	"github.com/sirupsen/logrus"
 )
 
 type ValidateOperatorBundlePolicy struct {
 }
 
-func (p ValidateOperatorBundlePolicy) Validate(bundle string) (bool, error) {
-	stdouterr, err := exec.Command("/usr/local/bin/operator-sdk", "bundle", "validate", "--verbose", bundle).CombinedOutput()
+func (p ValidateOperatorBundlePolicy) Validate(bundle string, logger *logrus.Logger) (bool, error) {
+	command, err := exec.Command("which", "operator-sdk").CombinedOutput()
 	if err != nil {
+		logger.Error("Error finding the operator-sdk command")
+		return false, nil
+	}
+	cmd := strings.TrimSuffix(string(command), "\n")
+	stdouterr, err := exec.Command(cmd, "bundle", "validate", "--verbose", bundle).CombinedOutput()
+	if err != nil {
+		logger.Error("Error will executing operator-sdk validate bundle")
```

Comment:
I think we better log `err` as well to help with the toubleshooting and not losing the context.

Reply:
Good point. I updated the logging messages.